README.md

```diff
@@ -64,4 +68,51 @@ We request that users of ACCESS-OM2 model [code](https://github.com/access-nri/a
 2. including an acknowledgement such as the following:
 
    *The authors thank the vibrant community of the Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA; [http://www.cosima.org.au](http://www.cosima.org.au)) for making the ACCESS-OM2 suite of models available at [https://github.com/COSIMA/access-om2](https://github.com/access-nri/access-om2).*
+
 3. let us know of any publications which use these models or data so we can add them to [our list](https://scholar.google.com/citations?hl=en&user=inVqu_4AAAAJ).
+
+
+## Citation
+
+If you use a recipe from the Cookbook for your research or teaching, or have based your analysis on one of the recipes, we would be grateful if you could cite:
+
+```bibtex@misc{cosima-cookbook,
+  author    = {Constantinou, Navid C. and
+               Hogg, Andy McC. and
+               Gibson, Angus and
+               Beucher, Romain and
+               Heerdegen, Aidan and
+               Steketee, Anton and
+               Holmes, Ryan M. and
+               Kiss, Andrew E. and
+               Morrison, Adele and
+               Fierro Arcos, Denisse and
+               Munroe, James and
+               Oliveira, Micael and
+               Yung, Claire and
+               Neme, Julia and
+               Ong, Ellie Q. Y. and
+               Doddridge, Edward and
+               Huneke, Wilma and
+               Schmidt, Christina and
+               Dias, Fabio Boeira and
+               Dawson, Hannah and
+               Jeffree, Jemma and
+               Bhagtani, Dhruv and
+               Vilela da Silva, Felipe and
+               Meijer, Jan Jaap and
+               Martínez Moreno, Josué and
+               White, Marc and
+               Auger, Matthis and
+               Moorman, Ruth and
+               Aguiar, Wilton and
+               Zika, Jan},
+  title     = {{COSIMA Cookbook: v0.1.0}},
+  month     = dec,
+  year      = 2024,
+  publisher = {Zenodo},
+  version   = {v0.1.0},
+  doi       = {10.5281/zenodo.14353853},
+  url       = {https://doi.org/10.5281/zenodo.14353853}
+}
```

Comment:
This will need to be updated every time we do a version bump or get a new contributor.
Plus, it's in two different places.
I'd prefer just to point to the Zenodo page and suggest getting the citation info there - that way the poor lost souls who don't use BibTex can export it in whichever other format they use.

Reply:
Regarding need to update every time: we can use https://doi.org/10.5281/zenodo.14353852 which will always point to the latest release

Reply:
That's great, and removing the list of authors from here means we don't need to constantly update it.

Reply:
Yes!

Reply:
@navidcy My 2 cents as I stumbled onto this PR: I'd recommend you add a `.zenodo.json` file here (in the GitHub repository) for the contibutors instead of manually editing the Zenodo file. This way, the Zenodo action automatically fetches the info, so you can get new contributors to add their own details here directly. It should populate the Zenodo archive automatically (instead of having to manually edit the contributors list on the Zenodo side). This is how, e.g., Plots.jl does it: [`.zenodo.json`](https://github.com/JuliaPlots/Plots.jl/blob/v1.40.9/.zenodo.json) and [Zenodo archive](https://zenodo.org/records/14094364) (and it has worked for me too, e.g., [`.zenodo.json`](https://github.com/JuliaOcean/AIBECS.jl/blob/master/.zenodo.json) and [Zenodo archive](https://zenodo.org/records/14241850)). You can probably directly export and copy the contents of your current JSON-formatted citation (maybe trim some of the fields). A list of the metadata fields you can enter in your `.zenodo.json` file is available [here](https://developers.zenodo.org/#representation).